libs/core/src/main/java/org/opensearch/core/common/io/stream/StreamOutput.java

```diff
@@ -1255,6 +1255,19 @@ public <E extends Enum<E>> void writeEnumSet(EnumSet<E> enumSet) throws IOExcept
         }
     }
 
+    /**
+     * Writes an optional EnumSet with type E that by serialized it based on it's ordinal value
```

Comment:
Similar to above, "it" implies singular; should be something like "each".

Reply:
done